src/interface/src/app/services/plan-notes.service.ts

```diff
@@ -10,43 +10,77 @@ export interface Note {
   created_at: string;
 }
 
+export interface NoteModel {
+  name: string;
+  urlMultiple: (modelObjectId: number) => string;
+  urlSingle: (modelObjectId: number, noteId: number) => string;
+}
+
+const noteModels: NoteModel[] = [
+  {
+    name: 'planning_area',
+    urlMultiple: (planningAreaId: number) =>
+      `/planning/planning_area/${planningAreaId}/note`,
+    urlSingle: (planningAreaId: number, noteId: number) =>
```

Comment:
I don't love having two urls here, but I think the formats of these could change, based on the model we're updating, so we can't assume the noteId will just be appended to the end.

Reply:
Im not sure if I understand this const, but if this is going to be basically a list of endpoints per model, maybe this should be a record
```
const noteEndpoints: Record<ModelName, {multipleUrl, singleUrl}>
```

Reply:
I wasn't sure if it might be more of a mess with string interpolation...but I updated it